### PR TITLE
Make AdjustTool Class Gobject-style constructed

### PR DESCRIPTION
**Pull request by ZenitsuDev**
Revises the AdjustTool class to follow gobject-style construction and the recommended code style.

### CODE REVIEW

File: src/editing_tools/AdjustTool.vala [hunk 1]
```diff
@@ -371,7 +419,8 @@ public class EditingTools.AdjustTool : EditingTool {
 
         /* set up temperature & tint */
         TemperatureTransformation temp_trans = (TemperatureTransformation)
```
Comment (ryonakano) [inline src/editing_tools/AdjustTool.vala:421]: Or we can use `var` for the left operand, like this:

```vala
var temp_trans = (TemperatureTransformation) transformations.get_transformation (
    PixelTransformationType.TEMPERATURE
);
```

File: src/editing_tools/AdjustTool.vala [hunk 2]
```diff
@@ -418,10 +470,11 @@ public class EditingTools.AdjustTool : EditingTool {
         virgin_histogram_pixbuf = histogram_pixbuf.copy ();
 
         DataCollection? owner = canvas.photo.get_membership ();
-        if (owner != null)
+        if (owner != null) {
             owner.items_altered.connect (on_photos_altered);
+        }
 
-        base.activate (canvas);
+        activate (canvas);
```
Comment (ryonakano) [inline src/editing_tools/AdjustTool.vala:477]: Hmm isn't the `base.` here necessary? This should call the original activate method of the base class

File: src/editing_tools/AdjustTool.vala [hunk 3]
```diff
@@ -447,16 +501,17 @@ public class EditingTools.AdjustTool : EditingTool {
         draw_to_pixbuf = null;
         fp_pixel_cache = null;
 
-        base.deactivate ();
+        deactivate ();
```
Comment (ryonakano) [inline src/editing_tools/AdjustTool.vala:504]: Here too

**Review by ryonakano — CHANGES_REQUESTED** [review]
Thank you for your contribution! Leaving some comments.

File: src/editing_tools/AdjustTool.vala (hunk 1 above)
Comment (ZenitsuDev) [reply src/editing_tools/AdjustTool.vala:421]: I think so too :) Thanks!

File: src/editing_tools/AdjustTool.vala (hunk 2 above)
Comment (ZenitsuDev) [reply src/editing_tools/AdjustTool.vala:477]: Thinking about it, yeah, probably, or the call would have been this.activate

**Comment by ZenitsuDev** [comment]
Konnichiwa @ryonakano -san, thanks for the comment! I'll be sure to improve this

**Review by ryonakano — APPROVED** [review]
Looks good, thank you!

**Pull request merged**